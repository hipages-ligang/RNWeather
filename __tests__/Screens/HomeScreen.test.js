import React from 'react';
import renderer from 'react-test-renderer';

describe('<HomeScreen />', () => {
    it('home screen snapshot', () => {
        const homeScreenTree = renderer.create(<homeScreen />).toJSON();
        expect(homeScreenTree).toMatchSnapshot();
    });
});
