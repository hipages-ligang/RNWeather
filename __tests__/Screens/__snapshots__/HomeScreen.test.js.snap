// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomeScreen /> home screen snapshot 1`] = `<homeScreen />`;
